### PR TITLE
FOUR-13453 order calcs list with drag and drop option

### PR DESCRIPTION
**Pull request by devmiguelangel**
## Issue & Reproduction Steps
As a designer, I want to have the option to Order the list of Calcs that I am creating in the Screen Builder.

## Related Tickets & Packages
[FOUR-13453](https://processmaker.atlassian.net/browse/FOUR-13453)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next

[FOUR-13453]: https://processmaker.atlassian.net/browse/FOUR-13453?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/computed-properties.vue [hunk 1]
```diff
@@ -15,50 +15,37 @@
...
+          btnDelete: { 'data-cy': 'calcs-table-remove' },
+        }"
+        @item-edit="editProperty"
+        @item-delete="deleteProperty"
+        @add-page="displayFormProperty"
+      >
+        <template #options="{ item }">
+          <button
+            v-b-tooltip="{ customClass: 'bypass-btn-tooltip' }"
+            :title="item.byPass ? 'Unbypass Calc' : 'Bypass Calc'"
```
Comment (caleeli) [inline src/components/computed-properties.vue:37]: Add translation for this labels

File: src/components/sortable/sortableList/SortableList.vue [hunk 2]
```diff
@@ -1,51 +1,85 @@
...
+          :key="field.key"
+          class="sortable-list-td sortable-list-header"
         >
+          {{ field.label }}
+        </div>
+        <div class="sortable-list-td"></div>
+      </div>
+      <div
+        v-for="(item, index) in sortedItems"
+        :key="index"
```
Comment (caleeli) [inline src/components/sortable/sortableList/SortableList.vue:21]: Consider to add a prefix to the :key like:
```suggestion
        :key="`item-${index}`"
```

File: src/components/computed-properties.vue (hunk 1 above)
Comment (devmiguelangel) [reply src/components/computed-properties.vue]: translations were added

File: src/components/sortable/sortableList/SortableList.vue (hunk 2 above)
Comment (devmiguelangel) [reply src/components/sortable/sortableList/SortableList.vue]: prefix was added

**Review by rodriquelca — APPROVED** [review]

**Pull request merged**